app/Livewire/Project/New/DockerImage.php

```
@@ -45,6 +45,10 @@ public function submit()
 
         $project = Project::where('uuid', $this->parameters['project_uuid'])->first();
         $environment = $project->load(['environments'])->environments->where('uuid', $this->parameters['environment_uuid'])->first();
+
+        // Determine the image tag based on whether it's a hash or regular tag
+        $imageTag = $parser->isImageHash() ? 'sha256-'.$parser->getTag() : $parser->getTag();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$imageTag'. (undefined)

(UnusedLocalVariable)